validation/distributed_simulations/distributed_geostrophic_adjustment.jl

```diff
@@ -80,7 +80,7 @@ end
 simulation.callbacks[:progress] = Callback(progress_message, IterationInterval(10))
 
 
-outputs = merge(model.velocities, model.free_surface)
+outputs = merge(model.velocities, model.free_surface.η)
```

Comment:
sorry I made a mistake here, if you use `merge` then also the free surface must be a namedtuple:

```suggestion
outputs = merge(model.velocities, (; η = model.free_surface.η))
```